engine/src/main/java/org/terasology/rendering/world/WorldRendererImpl.java

```diff
@@ -126,7 +127,7 @@
      * It's not, so for now, we use this factor to adjust for the disparity.
      */
     private static final float GROUND_PLANE_HEIGHT_DISPARITY = -0.7f;
-    private static RenderGraph renderGraph = new RenderGraph(); // TODO: Try making this non-static
+    private RenderGraph renderGraph = new RenderGraph();
```

Comment:
Okay, so this was a tricky thing. Not *strictly* necessary for exposing the API, but good to tackle nonetheless.
The way @RegisterSystem works is, it creates a new instance of the class (which is also used for the commands registered via @Command). Which means we effectively have 2 WorldRenederer instances - one that is used to draw the world, and one that is used for handling commands. Which is why i had to create the variable static. Definitely not ideal, and I think we should tackle it in this PR (since I'm tackling other render graph related things as well).

Reply:
WHAAAT??? So, we also have two render graphs? And which WorldRenderer instance is actually used for rendering? I mean, why is this not a massive problem?